tools/build-release.py

```diff
@@ -81,7 +81,7 @@ def setup_base_dir(release_top_path, base_path):
         org = os.path.join(release_top_path, file)
         dest = os.path.join(base_path, file)
         print("copying files: %s ===> %s" % (org, dest))
-        shutil.copyfile(org, dest)
```

Comment:
Why did you change that?

Reply:
copyfile will copy only the contents, but the metadata will be skipped, so the permissions will be skipped as well and set some default permissions.
copy2 will copy the metadata as well. see: https://docs.python.org/2/library/shutil.html

Reply:
Thanks for the clarification

Reply:
Thanks!